session.go

```diff
@@ -216,36 +244,48 @@ func (s *session) recvLoop() {
 }
 
 func (s *session) sendLoop() {
+	atomic.AddInt64(&sendLoops, 1)
+	defer func() {
+		atomic.AddInt64(&sendLoops, -1)
+	}()
+
 	for {
 		select {
 		case frame, more := <-s.out:
```

Comment:
I would say the same here -- just `open` or `chOpen` seems clearer than `more` to me.

Reply:
Fixed.